documentation/sidebars.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * * Creating a sidebar enables you to:
+ - create an ordered group of docs
+ - render a sidebar for each doc of that group
+ - provide next/previous navigation
+
+ The sidebars can be generated from the filesystem, or explicitly defined here.
+
+ Create as many sidebars as you want.
+ */
+
+module.exports = {
+    mySidebar: [
+        {
+            type: 'doc',
+            id: 'welcome',
+        },
+        {
+            type: 'category',
+            label: 'References',
+            items: [
+                {
+                    type: 'doc',
+                    id: 'configuration',
+                    label: 'Configuration',
+                },
+                {
+                    type: 'doc',
+                    id: 'api_reference',
+                    label: 'API Reference',
```

Comment:
Could you please move the files to the `/reference/` subfolder to mirror the ToC structure?